examples/complete-alb/main.tf

```diff
@@ -293,6 +303,14 @@ module "alb" {
       name_prefix                        = "l1-"
       target_type                        = "lambda"
       lambda_multi_value_headers_enabled = true
+      targets = [
+        {
+          target_id = "arn:aws:lambda:us-east-1:123456789012:function:lambda_function_1"
```

Comment:
Will it work if you specify non-existing ARN? If not, we need to update the code to create a real Lambda function and use it.

Reply:
^ bs